### ISSUE TITLE
flag to output only key=values in output command

### ISSUE THREAD

**Issue by cppforlife** [hashicorp/terraform#10454]
currently terraform output -json command includes extra namespacing around values so that additional metadata can be outputted. it would be nice to have a flag that would return flattened kv pairs. For example:

just `-json` flag:

```
{"key": { "value": 123, "sensitive": false, ... } }
```

with optional flag (`... -json -value-only`)

```
{"key": 123 }
```

wdyt? mind if i submit a PR for this feature?

**Comment by cppforlife** [comment]
cc @dpb587

**Comment by mitchellh** [comment]
Yes I'd be open to a PR for this. I may nitpick the CLI flags a bit but that'd be the easy part.

**Comment by rjinski** [comment]
Not sure if this is helpful but JSON manipulation with an [external tool](https://stedolan.github.io/jq/) might be able to solve this problem for you. 

```echo '{"foo": { "value": "bar"}, "bar": { "value": "foo"}}' | jq 'with_entries(.value |= .value)'```
